src/include/migraphx/quantization.hpp

```diff
@@ -23,12 +23,14 @@ std::size_t capture_arguments(program& prog,
                               const std::vector<std::string>& ins_names,
                               const std::function<void(std::size_t, std::vector<argument>)>& func);
 
-std::shared_ptr<std::vector<std::pair<float, float>>> capture_arguments_impl(
-    program& prog, const target& t, const std::vector<std::string>& ins_names = {"dot"});
+std::shared_ptr<std::vector<std::pair<float, float>>>
+capture_arguments_impl(program& prog,
+                       const target& t,
+                       const std::vector<std::string>& ins_names = {"dot", "convolution"});
```

Comment:
This is not needed for the impl function.

Reply:
we have a template function in this head file hat calls the impl function, so I think we need to have it here.